lib/gitea.ex

```diff
@@ -47,6 +47,35 @@ defmodule Gitea do
     Gitea.Http.post(url, params)
   end
 
+  @doc """
+  Create an organisation on Gitea.
+  The second argument opts is a keyword list value
+  and define the description, full_name and visibility
+  """
+  @spec remote_org_create(String.t(), list()) :: {:ok, map} | {:error, any}
+  def remote_org_create(org_name, opts \\ []) do
```

Comment:
any reason we don't just pass in a `Map` of `params` instead of the `org_name` and `opts` ? 💭

Reply:
I saw the `remote_org_create` function having only one required argument (the `org_name`) and the other values (description, full_name and visibility) as optional similar to the gitea api and the UI. In this case I think it makes sense to use a keyword list as it is used for this type of optional argument
see: https://elixir-lang.org/getting-started/keywords-and-maps.html#keyword-lists
> Keyword lists are a data-structure used to pass options to functions

However if you consider that all the values are required we can pass a Map (or even define a struct).

Let me know if you prefer the "map" version of the function

Reply:
yeah, sadly, `remote_org_create/1` in `Gogs` didn't work https://github.com/dwyl/gogs/issues/8#issuecomment-1115464858 so I didn't pursue it ... 
I find keyword lists cumbersome.
in this case having a _single_ `params` argument with all the necessary fields (only `params.username` is required) will make it immediately clear to the person using the app that their `params` are passed in directly.